modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -10459,4 +10438,38 @@ void releaseAccessible() {
         }
     }
 
+    /**
+     * Requests focus traversal from this {@code Node} in the specified direction.
+     * A successful traversal results in the newly focused {@code Node} visibly indicating its focused state.
+     *
+     * @param direction the direction of focus traversal
+     * @return true if traversal was successful
+     * @since 24
+     */
+    public final boolean requestFocusTraversal(TraversalDirection direction) {
```

Comment:
possible alternative names:

moveFocus
requestFocusTraversal
tryToFocus
tryMoveFocus

Reply:
Of these, I like `requestFocusTraversal` best. Another alternative: `requestMoveFocus`.

Reply:
Plain `requestFocus` can also work as it can be overloaded with the direction parameter.

Reply:
I like the overloaded `requestFocus` idea as well.

Reply:
I really dislike overloading in this case - the semantics of these two methods is totally different: one moves the focus to the node, one moves the focus somewhere else.